COMET.Web.Common/Components/SingleIterationApplicationTemplate.razor

```diff
@@ -41,6 +41,7 @@ else
 		<div class="row m-top-10px">
 			@if (this.ViewModel.SelectedIteration is not null)
 			{
+				<DxTextBox ReadOnly="true" Text="@($"Model and Iteration : {this.ViewModel.SelectedIterationData.IterationName}")" SizeMode="SizeMode.Large"></DxTextBox>
```

Comment:
i propose to rename from "Model and Iteration" to "Active Model"

also the spacing between the title box and the next controls on the screenshot needs improvement, they are too close to eachother